extensions/amp-video/0.1/amp-video.js

```diff
@@ -307,16 +332,17 @@ class AmpVideo extends AMP.BaseElement {
   /** @override */
   getMetadata() {
     const poster = this.element.getAttribute('poster');
-    if (poster) {
-      return {
-        'title': '',
-        'artist': '',
-        'album': '',
-        'artwork': [
-          {'src': poster},
-        ],
-      };
-    }
+    const artist = this.element.getAttribute('artist');
+    const title = this.element.getAttribute('title');
```

Comment:
this should now move to `buildCallback` and fill the `this.metadata_` there. Here we just return `this.metadata_` (otherwise all the bind support added won't work)